.github/workflows/ci.yaml

```diff
@@ -22,15 +22,18 @@ jobs:
   build:
     needs: ci
     runs-on: ubuntu-latest
+    env:
+      TRAVIS_BRANCH: ${{ github.head_ref }}
+      TRAVIS_PULL_REQUEST: ${{ github.event.number }}
```

Comment:
I moved this up to the global section only because we have the other one here. I'd rather leave it in the specific step, but then it's defined in 2 places, which is gross.